src/components/Footer.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+
+const Footer = () => {
+    return (
+        <footer className="d-flex flex-wrap justify-content-between align-items-center py-3 my-4 border-top">
```

Comment:
Do we really want to use https://getbootstrap.com/ instead of https://mui.com/ library? What are the cons and pros? Do you think it will be much faster to implement this with bootstrap? @DevilWillReign 🤔

Reply:
I will change it to MUI, i am more used to bootstrap but will change singce it looks like its a little lighter